pinot-core/src/test/java/org/apache/pinot/queries/ExplainPlanQueriesTest.java

```diff
@@ -284,7 +285,7 @@ public void setUp()
       tableDataManager.addSegment((ImmutableSegment) indexSegment);
     }
     InstanceDataManager instanceDataManager = mock(InstanceDataManager.class);
-    when(instanceDataManager.getTableDataManager(RAW_TABLE_NAME)).thenReturn(tableDataManager);
+    when(instanceDataManager.getTableDataManager(OFFLINE_TABLE_NAME)).thenReturn(tableDataManager);
```

Comment:
This change is breaking the test. QueryExecutor is getting table name without type as argument and hence gets no tableDataManager

Reply:
This test was not well implemented because we should use table name with type on the server side. Fixed the test